domenetjenester/brevbestiller/src/test/java/no/nav/foreldrepenger/melding/datamapper/brev/InnhentOpplysningerBrevMapperTest.java

```diff
@@ -81,12 +81,28 @@ public void setup() {
     }
 
     @Test
-    public void skal_velge_mottatt_dato_fra_endringssøknad_når_endring() {
+    public void skal_velge_mottattt_dato_fra_siste_søknad_mottatt_når_flere() {
+        LocalDate andreJanuar = LocalDate.of(2019, 1, 2);
+        LocalDate fjerdeJanuar = LocalDate.of(2019, 1, 4);
```

Comment:
Her ville jeg lagt inn en tredje dato (og en tredje søknad), og sørget for at den nyeste (som du ønsker å få returnert) er nr 2 av 3 som legges til. Da sikrer du at det må sorteres riktig for at du skal kunne få den returnert.

Reply:
Done